portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Map</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAk7hZS_MGp70LtNVUBQ6Gm3aGrfpsNtmU&callback=initMap&libraries=&v=weekly"
```

Comment:
Are API keys okay to be pushed to Github?

Reply:
No, they shouldn't be pushed to github.  There are definitely people that scrape github for api keys (this happened to a group while we were in school). Let me try to find some resources for how to avoid doing this.

Reply:
Just deleted this one to be safe!